modules/remotebackend/regression-tests/test-schema.sql

```diff
@@ -64,23 +64,23 @@ create table tsigkeys (
 
 create unique index namealgoindex on tsigkeys(name, algorithm);
 
-insert into domains (name,type) VALUES('example.com','NATIVE');
-insert into domains (name,type) VALUES('up.example.com','NATIVE');
+insert into domains (name,type) VALUES('example.com.','NATIVE');
+insert into domains (name,type) VALUES('up.example.com.','NATIVE');
```

Comment:
Am I right in understanding that before this change, your schema and data were gsql-compatible, but this changes here?